dali/pipeline/operators/displacement/displacement_filter_impl_cpu.h

```diff
@@ -110,6 +112,67 @@ class DisplacementFilter<CPUBackend, Displacement,
   USE_OPERATOR_MEMBERS();
 
  private:
+  // TODO(klecki) We could probably interpolate with something other than float,
+  // for example integer type
+  struct LinearCoefs {
+    LinearCoefs(float rx, float ry) : rx(rx), ry(ry), mrx(1.f - rx), mry(1.f - ry) {}\
+    LinearCoefs() = default;
+    float rx, ry, mrx, mry;
+  };
+
+  template <typename T>
+  bool ShouldTransform(Point<T> p) {
```

Comment:
argument can be a `const &`

Reply:
This is rather simple type, that could also be passed by value. Still, there probably won't be any noticeable difference.

Reply:
I would prefer to use pass-by-value semantics here.